building/glass/glass.py

```diff
@@ -43,16 +45,16 @@
     if args.clean or args.rebuild:
         any = False
         for p in projects:
-            any |= p.clean(args.branch)
+            any |= p.clean(branch_config.name)
```

Comment:
It feels a little bit weird to pass a branch config name to `clean`, and a branch config to `run`. Maybe these should both take a full config?